src/main/java/uk/gov/hmcts/dm/service/SecurityUtilService.java

```diff
@@ -48,7 +49,10 @@ public String getCurrentlyAuthenticatedServiceName() {
     }
 
     public static HashSet<String> sanitizedSetFrom(Collection<String> collection) {
-        return collection.stream().map(String::trim).collect(Collectors.toCollection(HashSet::new));
+        return collection.stream()
+            .map(String::trim)
+            .map(StringUtils::convertValidLogString)
```

Comment:
this function is used to trim user/document roles, is this not interfering with that?

Reply:
Yes. It is. So that was another reason to get it reviewed by you. But this change should not break it. Your view?